spec/components/sdg/goals/plain_tag_list_component_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+describe SDG::Goals::PlainTagListComponent, type: :component do
+  let(:debate) { create(:debate, sdg_goals: [SDG::Goal[1], SDG::Goal[3]]) }
+  let(:component) { SDG::Goals::TagListComponent.new(debate) }
```

Comment:
These tests aren't testing the `PlainTagListComponent` but ghe `TagListComponent`.
```suggestion
  let(:component) { SDG::Goals::PlainTagListComponent.new(debate) }
```